src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,12 @@
+
 package core.basesyntax;
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        for (int i = 0; i < 3; i++) {
```

Comment:
Do not use magic numbers, make a constants!

Reply:
what is mean magic numbers?

Reply:
Before requesting a review, please go through all `Common mistakes` provided at the end of the task description:
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#all-magic-numbers-in-your-code-should-be-constants